safe-global/safe-react
# add: safe token widget to app header

## What it solves
Adds token widget to app header on mainnet and rinkeby.

## How to test it
- Open a Safe on mainnet or rinkeby
- observe header displaying current balance of safe tokens

## Analytics changes
- none

## Screenshots
![localhost_3000_rin_0x979774d85274A5F63C85786aC4Fa54B9A4f391c2_apps_appUrl=https___safe-claiming-app pages dev_](https://user-images.githubusercontent.com/2670790/188448131-222fa760-8df0-4ec0-8f29-4d3572859b59.png)
